src/views/ForgotPasswordPageChange/index.js

```diff
@@ -5,113 +5,142 @@ import Button from "react-bootstrap/Button";
-      document.getElementsByClassName("move=btn")[0].disabled = true;
-    }
-  }
-
   handleChange = event => {
-    this.setState({[event.target.name]: event.target.value});
+    const { info, value } = event.target;
+
+    this.setState({
+      [info]: value
+    });
+    console.log(this.state);
   };
 
   handleSubmit(event) {
     event.preventDefault();
-
-    const { code } = this.props.location;
-    const { email } = this.props.location;
-
-    var dataPassword = {
-      Name: 'password',
-      Value: this.state.password
-    };
-    var dataConfirm = {
-      Name: 'confirmpassword',
-      Value: this.state.confirmpassword
-    }
-
-    if (dataPassword.Value === dataConfirm.Value) {
-      var params = {
-        ClientId: process.env.REACT_APP_COGNITO_CLIENT_ID,
-        ConfirmationCode: code,
-        Password: dataPassword.Value,
-        Username: email
-      };
-      cognitoIdentityServiceProvider.confirmForgotPassword(params, function(err, data) {
-        if (err) {
-          document.getElementById("display_error").innerHTML = err;
-          document.getElementById("display_error").style.color = "#ff0000";
-        }
-        else {
-          document.getElementById("display_error").innerHTML = "Your password has been successfully changed.";
-        }       
-      });
-    } else {
-      document.getElementById("display_error").innerHTML = "Please ensure the two passwords entered match.";
-      document.getElementById("display_error").style.color = "#ff0000";
-    }  
+    console.log(this.state);
   }
   render() {
     return (
-      <Form 
-        className="inputForm"
-        onPointerMove={this.handleNoCode}
-      >
-        <Form.Group controlId="inputForm.password">
-          <Form.Label>Password</Form.Label>
-          <Form.Control
-            required
-            name="password"
-            type="password"
-            placeholder=""
-            onChange={this.handleChange}
-          />
-        </Form.Group>
-        <Form.Group controlId="inputForm.confirmPassword">
-          <Form.Label>Confirm Password</Form.Label>
-          <Form.Control
-            required
-            name="confirmpassword"
-            type="password"
-            placeholder=""
-            onChange={this.handleChange}
-          />
-        </Form.Group>
-        <Button
-          className="move=btn"
-          variant="success"
-          type="submit"
-          onClick={this.handleSubmit.bind(this)}
-        >
-          <Link className="btn-link" to="/forgotpassword">
-            Change Password
-          </Link>
-        </Button>
-        <div className="display-error" id="display_error"></div>
-        <Route path="/LoginPage">
-          <LoginPage />
-        </Route>
-      </Form>
+      <div>
+        <div style={{ backgroundColor: "#EEEEEE", margin: "auto", borderRadius: "16px", overflow: "hidden", boxShadow: " 0 19px 38px rgba(0, 0, 0, 0.30), 0 5px 5px rgba(0, 0, 0, 0.22)" }}>
```

Comment:
Move all styles into a class in`/style.js` then import them at the top with `import { style } from './style'` so you can do things like `<div style={style.background}>`. Do this for all the inline styles you have here as well as the css classes you defined below!

Reply:
Also, let's define colours so we don't have to keep copy/pasting the hex codes. Create a new folder `src/assets/colours` and create a file inside it called index.js that has all the colours defined like so: `export default { darkBlue: '#EEEEEE' }` and then in the styles.js file when you want to use this colour, import the file like so: `import { colours } from '../../../assets/colours'` and reference the colour as `colors.darkBlue`